README.md

```diff
@@ -284,118 +270,17 @@ LOG_NationalCommunityChildHealthDatabase(NCCHD)_CHILD_2024-04-05-14-37-36.csv
--   the lookup file only works for 1:1 mappings right now, i.e. the DataElement should only be listed once in the lookup file
-
-#### output dir: 
-
-- the path to the directory where the two csv output files will be saved. By default, the current working directory is used
-
-#### table_copy:
-
--   default is TRUE, so set this to FALSE if you want to deactivate table copying
-- table copying means that the categorisations you made for previous tables will be carried over to this table, as long as the csv files share an output_dir
--   this can be useful because the same data elements (variables) appear across multiple tables within one dataset
--   copying from one table to the next will save the user time, and ensure consistency of categorisations across tables
--   the 'Note' column in the output csv file will indicate that the categorisation has been copied and where from
--   a typical session could look like this:
-
-*Run 1, select table 'CHILD'*
-
-```         
-ℹ Processing Table 6 of 13
-
-── Table Name ──
-
-CHILD
-
-── Table Last Updated ──
-
-[datetime]
-```
-
-*Run 2, select table 'CHILD_BIRTHS' (the function notices we have already run the table 'CHILD')*
-
-```
-ℹ Processing Table 7 of 13
-
-── Table Name ──
-
-CHILD_BIRTHS
-
-── Table Last Updated ──
-
-[datetime]
-...
-ℹ Copying from previous session(s): 
-
-[1] "OUTPUT_NationalCommunityChildHealthDatabase(NCCHD)_CHILD_[datetime].csv"
-
-```
-
-*Run 3, select table 'PATH_BLOOD_TESTS' (the function notices we have already run the table 'CHILD' and 'CHILD_BIRTHS')*
-
-```
-ℹ Processing Table 8 of 13
-
-── Table Name ──
-
-PATH_BLOOD_TESTS
-
-── Table Last Updated ──
+### Tips and potential next steps
 
-[datetime]
-...
-ℹ Copying from previous session(s): 
-
-[1] "OUTPUT_NationalCommunityChildHealthDatabase(NCCHD)_CHILD_[datetime].csv"
-[2] "OUTPUT_NationalCommunityChildHealthDatabase(NCCHD)_CHILD_BIRTHS_[datetime].csv"
-
-```
-
-*And so on ...* Each run where you process a table has the potential to be shorter for the user to complete because if there are the same data elements that appear across tables, the user will not be asked to categorise them twice.
-
-### Potential use-cases for the output files
-
-The csv output file containing the categorisation for each data element could be used as an input in later analysis steps to filter variables and visualise how each variable maps to research domains of interest.
-
-Categorisations across researchers can be compared by using the function [R/compare_sessions.R](R/compare_sessions.R). Type `?compare_sessions` to read the manual on how to run this function. In brief, it compares csv outputs from two sessions, finds their differences, and asks for a consensus.
+- First thing to do next is to try `browseMetadata` and `mapMetdata` with your own inputs, without the demo mode! For detailed information on each input argument, reference the manual by typing `?mapMetadata`.
```

Comment:
The main reasons not to run in demo mode:
- input your json file of choice (for example, the demo mode only processed on example dataset, NCCHD) 
- process more than 20 data elements (variables) at once
- consider using their own lookup table if they have exiting info on mapping 
Does that make sense? If yes, we can try to expand the text

Reply:
```suggestion
- First thing to do next is to try `browseMetadata` and `mapMetdata` with your own inputs, without the demo mode! This would allow you to: input a metadata json file of your choice, process more than 20 data elements at once and use your own look-up table. For detailed information on each input argument, remember to reference the manual by typing `?mapMetadata`.
```

Reply:
```suggestion
- First thing to do next is to try `mapMetdata` without the demo mode! You can still use the example demo files, but this will allow you to process more than 20 data elements:
`demo_json_file <- system.file("inputs/national_community_child_health_database_(ncchd)_20240405T130125.json", package = "browseMetadata")`
`demo_domains_file <- system.file("inputs/domain_list_demo.csv",package = "browseMetadata")`
`browseMetadata(json_file = demo_domains_file, domain_file = = demo_domains_file,)`
- Next thing to try is to change these json_file and domain_file inputs! If you change the json_file, remember to run `browseMetdata` before running `mapMetdata` with this new file. There are also other options for this function, type `?mapMetdata` to find out more. 
```

Reply:
@Rainiefantasy how's this? https://github.com/aim-rsf/browseMetadata/commit/cfb636b561352a76252b92e9f5f8bf850782b8bf